recipes/snappy/meta.yaml

```diff
@@ -0,0 +1,25 @@
+{% set version = "1.1.3" %}
+
+package:
+  name: snappy
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/google/snappy.git
+  git_tag: {{ version }}
+
+requirements:
+  build:
+    - msinttypes  # [win]
+    - libtool # [unix]
+    - automake # [unix]
```

Comment:
Also, selectors.

* 2 spaces after package name (minimum)
* line up indentation of these selectors.